documents/decimal.go

```diff
@@ -154,6 +154,16 @@ func (d *Decimal) SetBytes(dec []byte) error {
 	return d.SetString(s)
 }
 
+// NewDecimal returns a new decimal from given string
```

Comment:
isn't this function `utils` material?

Reply:
The `Decimal` is defined in document package. If we move this utils, it will have to import from `documents` causing all sorts of dep cycles